src/services/submissionService.ts

```diff
@@ -186,17 +224,26 @@ export const mapGqlSearchFiltersToDbSearchFilters = (filters: gqlType.Submission
     const filteredLanguages = (mappedLanguages ?? [])
         .filter(lang => lang !== undefined) as dbSchema.SpokenLanguage[]
 
+    const mappedOrderBy = filters.orderBy ? filters.orderBy
+        .filter((o): o is gqlType.OrderBy => Boolean(o && o.fieldToOrder && o.orderDirection))
+        .map(order => ({
+            fieldToOrder: order.fieldToOrder as string,
```

Comment:
Future feature: In the future we'll need to validate they're sending in valid fields that exist and it isn't malicious JavaScript in string format.